src/plugins/saved_objects_finder/kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "savedObjectsFinder",
+  "owner": {
```

Comment:
that's a good question, we're consumers and creators of the plugin but it's not our domain, wdyt @pgayvallet. CODEOWNERS also needs to be modified

Reply:
`These should either be maintained by the wider Kibana analytics team (the main consumer) or shared UX team.`

https://github.com/elastic/kibana/issues/46435#issuecomment-1038898869

the way I read it is that it should be move out of core ownership :)

Reply:
In that case do you think we should change it to Data Discovery, or is there another team that would be better fit? And thanks, I'll make sure to update the CODEOWNERS file when we decide.

Reply:
Yea, the long (at the time) / mid (hopefully now) term goal was to get rid of the whole SO client-side plugin and have the various components be dispatched to the proper (TBD) teams. 

So unless you have any objection, I think it's best if you get ownership of this new plugin. Or, if you think it may have more consumers anytime soon, ask shared-ui if they would be fine owning it.